identity/client/src/main/java/com/azure/identity/IdentityClientOptions.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.http.ProxyOptions;
+
+import java.util.function.Function;
+
+/**
+ * Options to configure the IdentityClient.
+ */
+public final class IdentityClientOptions {
+    private static final String DEFAULT_AUTHORITY_HOST = "https://login.microsoftonline.com/";
+    private static final int MAX_RETRY_DEFAULT_LIMIT = 20;
```

Comment:
>private static final int MAX_RETRY_DEFAULT_LIMIT = 20; [](start = 4, length = 54)

Is this the normal retry limit?  It seems very high.

Reply:
Turned down to 3. Copied from the previous MSI implementation. They had an interesting Fibonacci retry logic in there.